wrappers/cabal.hs

```diff
@@ -10,9 +10,11 @@ main = do
   case args of
     "--interactive":_ -> do
       getCurrentDirectory >>= putStrLn
-      -- note this probably breaks if paths have spaces in
-      putStrLn $ unwords args
+      putStrLn $ delimited args
     _ -> do
       ph <- spawnProcess "ghc" args
       code <- waitForProcess ph
       exitWith code
+
+delimited :: [String] -> String
+delimited = (++) "-here " . foldr (\a as -> a ++ '\NUL' : as) ""
```

Comment:
What is this `-here` for?

Reply:
That is a piece of debugging output which I have accidentally left in my pull request. :sweat_smile: I will fix that right away!